nf_core/pipeline-template/nextflow.config

```diff
@@ -216,6 +217,9 @@ env {
 // Capture exit codes from upstream processes when piping
 process.shell = ['/bin/bash', '-euo', 'pipefail']
 
+// Disable process selector warnings by default. Use debug profile to enable warnings.
+nextflow.enable.configProcessNamesValidation = false
```

Comment:
No need to add this param in the global scope

Reply:
Thanks for the review!
* I've added these notes to the website - does it look okay? https://github.com/nf-core/website/pull/2040/files
* I will work on CONTRIBUTING.md

Reply:
> No need to add this param in the global scope

The default for this setting in nextflow is true. My idea for this is to set it to false by default, then return it to true if using debug.

Reply:
yes, sorry I missed that it was `true` by default. Your code looks good then